index.css

```diff
@@ -0,0 +1,218 @@
+body {
+    background-color:#1b1b2f;
+    width: auto;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+/* navigation */
+
+header {
+    background-color: #1f4068;
+    color: beige;
+    text-align: center;
+    padding-top: 20px;
+}
+nav {
+    width: auto;
+}
+
+nav a {
+    text-decoration: none;
+    color: beige;
+}
+
+nav a:visited{
+    color: beige;
+}
+
+nav a:hover{
+    color: #e43f5a;
+}
+
+nav h2 {
+    width: 25%;
+    float: left;
+    vertical-align: top;
+    margin-top: 0px;
+    font-weight: bolder;
+}
+
+ul.navbar-list {
+    margin-top: 30px;
+}
+
+ul {
+    /* width: 75%; */
+    margin-left: 35%;
+    padding-bottom: 50px;
+    /* float: top; */
+    /* vertical-align: top; */
```

Comment:
Avoid leaving code which has been commented out; this can result in someone later uncommenting the line, introducing unintended (and often difficult to debug) consequences.